src/views/RatingCalculator/test_gor.ts

```diff
@@ -1,84 +0,0 @@
-/*
```

Comment:
Unfortunately git didn't detect this was a move, but this was renamed to gor.test.ts